app/views/admin/gias/school_changes/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :title, "Changes to #{@school.name_and_urn}" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_gias_school_changes_path) %>
+
+<span class="govuk-caption-l"><%= @school.name_and_urn %></span>
+<h1 class="govuk-heading-xl">School details changes</h1>
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Attribute</th>
+      <th scope="col" class="govuk-table__header">Current value</th>
+      <th scope="col" class="govuk-table__header">New value</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @school.attributes.except("id", "created_at", "updated_at", "urn").each do |k,v| %>
+      <% current_value = @school.counterpart.send(k) %>
+      <% if current_value != v %>
+        <tr class="govuk-table__row">
+          <th scope="row" class="govuk-table__header"><%= k.humanize %></th>
+          <td class="govuk-table__cell"><%= current_value %></td>
+          <td class="govuk-table__cell">
+              <strong><i><%= v %></i></strong>
```

Comment:
`i` doesn't really mean italic anymore, this would probably be best off replaced with `<em>` or a classname (preferably a classname)

Reply:
Yes, good catch, you're right , I couldn't remember `em` last night when I added that!